manipulate_pim_data/teamwork_assistant/calculation_step.rst

```diff
@@ -2,7 +2,7 @@ Add A Calculation Step
 ======================
 
 To add a new step you just need to create a new Step Class that implements
-`PimEnterprise\Component\ActivityManager\Job\ProjectCalculation\CalculationStep\CalculationStepInterface` interface.
+`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep` interface.
```

Comment:
```suggestion
`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStepInterface`.
```

Reply:
It's `Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep\\CalculationStepInterface`, there is a folder `CalculationStep`.

Reply:
@tseho we should have synchronized :joy: